tests/input/input.js

```diff
@@ -139,16 +136,24 @@ async function simulateDeletePress(input) {
 }
 
 // eslint-disable-next-line react/prefer-stateless-function
-class ClassInputComponent extends React.Component {
+class InnerClassInputComponent extends React.Component {
   render() {
+    // eslint-disable-next-line react/prop-types
+    const { innerRef, ...restProps } = this.props;
     return (
       <div>
-        <input {...this.props} />
+        <input ref={innerRef} {...restProps} />
       </div>
     );
   }
 }
```

Comment:
Doing a test with the new style of wrapping a class component in a `React.forwardRef`